README.md

```diff
@@ -4,6 +4,13 @@ easy-pdf-merge is a node module to merge multiple PDFs into a single PDF easily.
 
 Currently only callback style is supported. That is only Async functions are supported.
 
+## Install :
```

Comment:
Can we have two sections ? One saying "CLI Usage" and another saying "API Usage" ?

So, let all my stuff come under "API Usage", which is presently named as "Usage". 
Let's keep the "CLI Usage" section on top! And put the "Install" section and "Command usage" in it together.
Because it looks scattered now. So, what do you think ?